src/mkdocs_include_markdown_plugin/event.py

```diff
@@ -8,8 +8,10 @@
 import os
 import re
 import textwrap
+import requests
```

Comment:
You are including requests without adding it as dependency to *pyproject.toml*. I prefer to use of [`urllib`](https://docs.python.org/3/library/urllib.html) directly for this trivial case of downloading plain text avoiding the installation of a dependency.

Reply:
fixed, please review